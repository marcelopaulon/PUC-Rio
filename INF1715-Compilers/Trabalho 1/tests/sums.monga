int main(void) {
	int raynan = 1412122;
	int marcelo = 1411029;
        double b = 0.0001;

        double soma = raynan + marcelo + 0.5599999 + b;

        return soma;
}


