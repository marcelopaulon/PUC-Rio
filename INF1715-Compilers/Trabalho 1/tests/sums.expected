START
264(1)
269(1)main
40(1)
267(1)
41(1)
123(1)
264(2)
269(2)raynan
61(2)
272(2)1412122
59(2)
264(3)
269(3)marcelo
61(3)
272(3)1411029
59(3)
269(4)double
269(4)b
61(4)
271(4)0x1.a36e2eb1c432dp-14 / 0.000100
59(4)
269(6)double
269(6)soma
61(6)
269(6)raynan
43(6)
269(6)marcelo
43(6)
271(6)0x1.1eb84e9086cb6p-1 / 0.560000
43(6)
269(6)b
59(6)
266(8)
269(8)soma
59(8)
125(9)
END
